README.md

```diff
@@ -22,6 +27,7 @@ usage: elicznik [-h] [--format {table,csv}] [--api {chart,csv}] username passwor
 positional arguments:
   username              tauron-dystrybucja.pl user name
   password              tauron-dystrybucja.pl password
+  site                  tauron-dystrybucja site. XXXXXXXXX_YYYYYYYY_ZZZZZZ format. 
```

Comment:
There are no corresponding changes in `__main__.py` that would add support for the new parameter.   

Maybe you forgot to add changes in that file to the commit?

Reply:
I did :see_no_evil: